netplay/joinroom.cpp

```diff
@@ -362,3 +366,13 @@ void JoinRoom::processBinaryMessage(QByteArray message)
         }
     }
 }
+
+void JoinRoom::updatePing(quint64 elapsedTime, const QByteArray&)
+{
+    pingLabel->setText("Ping: " + QString::number(elapsedTime) + " ms");
+}
+
+void JoinRoom::sendPing()
+{
+    webSocket->ping();
+}
```

Comment:
Really minor thing ,can you add a newline at the end of the file? If there is no newline it can look odd when printing it in linux

Reply:
👍  done